### DISCUSSION TITLE
Commented out code in CoreNodeRenderer.java

### DISCUSSION THREAD

**Discussion by kirkilj** [jelovirt/org.lwdita#154] [Q&A]
Starting where the clean() method is defined, there are several dozen lines of commented out code. Some of it deals with metadata, which is one of the areas we may need to customize, because we may prefer to have <othermeta> generated instead of <data> for non-standard Yaml Header info. Once we specialize some additional DITA metadata elements, we my want to render those specialized metadata elements instead.

Are these commented out lines for debug purposes or are they development leftovers?

The render() method also has some of these lines.

**Comment by jelovirt** [comment]
They are unused code that I haven't removed. Because I have been working on the codebase alone, I have been able to keep trash like that around, but I really should remove them.

**Reply by kirkilj** [reply]
No worries. Thanks for the clarification.
